packages/http-client-js/src/components/operation-handlers/paging/request-send.tsx

```diff
@@ -13,42 +18,96 @@ import {
   getOperationParameters,
 } from "../../operation-parameters.jsx";
 
+/**
+ * Properties for the HttpRequestSend component.
+ * @param httpOperation The HTTP operation to be performed.
```

Comment:
is the `@param` correct here? i notice somewhere else is `@property`. any difference?